### PR TITLE
Fix for exports_left, fix for some edge cases [SCI-3345]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-3345](https://biosistemika.atlassian.net/browse/SCI-3345)

### What was done
- Fix for showing `reports_left`, don't care about reports if are older than today
- Add some tests for edge cases

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -529,13 +530,20 @@ def increase_daily_exports_counter!
   end
 
   def has_available_exports?
-    limit = (Rails.application.secrets.export_all_limit_24h || 3).to_i
-    last_export = export_vars[:last_export_timestamp]
+    last_export = export_vars[:last_export_timestamp] || 0
```
Comment (radiokills) [inline app/models/user.rb]: I know this is a bit nitty gritty, but wouldn't it make more sense to name this variable `last_export_timestamp`. The way it is named now it implies we're dealing with whole export, not just timestamp.

Comment (radiokills) [reply app/models/user.rb]: Or maybe last_export_at

File: app/controllers/teams_controller.rb [hunk 2]
```diff
@@ -251,23 +251,21 @@ def export_projects
...
+                      num_of_requests_left: current_user.exports_left }
           ),
           title: t('projects.export_projects.modal_title')
         }
       else
         render json: {
           html: render_to_string(
             partial: 'projects/export/error.html.erb',
-            locals: { limit: limit }
+            locals: { limit: TeamZipExport.exports_limit }
```
Comment (radiokills) [inline app/controllers/teams_controller.rb]: :+1:
